src/patternfly/components/Drawer/drawer-example-resizable-panel.hbs

```diff
@@ -0,0 +1,12 @@
+{{#> drawer-panel drawer-panel--NoBody="true"}}
```

Comment:
This was bugging me that we have `drawer-panel--NoBody` set and are including `{{#> drawer-body}}` below. I changed the logic of `drawer-panel.hbs` a little to see if we can make that file a little more logical and added a PR for your branch here - https://github.com/DaoDaoNoCode/patternfly/pull/1. Let me know what you think, if you think that improves it or you'd like to handle it a different way.

Reply:
@mcoker Thank you for your suggestions! Actually I had no idea about how to organize the logic at first but it's much more clear to refer to your commit. I will update this and other examples that using `drawer-panel--NoBody` in the doc.